android/src/main/java/com/swmansion/rnscreens/Screen.kt

```diff
@@ -137,7 +137,7 @@ class Screen(
         r: Int,
         b: Int,
     ) {
-        if (container is ScreenStack && changed) {
+        if (changed) {
```

Comment:
It was added since the logic inside is intended only for `native-stack`. So I have no idea why removing it would change anything. Can we have more information?

Reply:
@WoLewicki I updated the description with some more information. That's all I got, I'll keep on investigating.